en/restore-from-aws-s3-using-br.md

```diff
@@ -0,0 +1,276 @@
+category: how-to
+---
+
+# Restore Data from AWS S3 using BR
+
+This document describes how to restore the TiDB cluster data backed up using TiDB Operator in Kubernetes. [`BR`](https://pingcap.com/docs/v3.1/reference/tools/br/br) is used to perform the restoration.
+
+The restoration method described in this document is implemented based on Custom Resource Definition (CRD) in TiDB Operator v1.1 or later versions.
+
+This document shows an example in which the backup data stored in the specified path on the AWS S3 storage is restored to the TiDB cluster.
+
+## Three methods to grant AWS account permissions
+
+Refer to [Back up Data to AWS S3 using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions).
+
+## Prerequisites
+
+### Grant permissions by importing AccessKey and SecretKey
+
+1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml), and execute the following command to create the role-based access control (RBAC) resources in the `test2` namespace:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test2
+    ```
+
+2. Create the `s3-secret` secret which stores the credential used to access the S3-compatible storage:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-literal=secret_key=yyy --namespace=test2
+    ```
+
+3. Create the `restore-demo2-tidb-secret` secret which stores the account and password needed to access the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic restore-demo2-tidb-secret --from-literal=password=<password> --namespace=test2
+    ```
+
+### Grant permissions by associating IAM with Pod
+
+1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml), and execute the following command to create the role-based access control (RBAC) resources in the `test2` namespace:
+
+     {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test2
+    ```
+
+2. Create the `restore-demo2-tidb-secret` secret which stores the account and password needed to access the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic restore-demo2-tidb-secret --from-literal=password=<password> --namespace=test2
+    ```
+
+3. Create the IAM role:
+
+    - To create a IAM role for the account, refer to [Create an IAM User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html).
+    - Give the IAM role you created the required permission (refer to [access policies manage](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html) for details). Because `Restore` needs to access AWS S3 storage, IAM is given the `AmazonS3FullAccess` permission.
```

Comment:
```suggestion
    - Give the required permission to the IAM role you have created  (refer to [access policies manage](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html) for details). Because `Restore` needs to access the AWS S3 storage, the IAM here is given the `AmazonS3FullAccess` permission.
```